File: src/net/zomis/tttultimate/games/TTControllers.java
package net.zomis.tttultimate.games;

import net.zomis.tttultimate.TTFactories;


public class TTControllers {

	public static TTController connectFour() {
		return new TTClassicControllerWithGravity(new TTFactories().classicMNK(7, 6, 4));
	}

	public static TTController classicTTT() {
		return new TTClassicController(new TTFactories().classicMNK(3));
	}

	public static TTController ultimateTTT() {
		return new TTUltimateController(new TTFactories().ultimate());
	}

}

File: uttt-common/src/main/kotlin/net/zomis/tttultimate/games/TTControllers.kt
package net.zomis.tttultimate.games

import net.zomis.tttultimate.TTFactories

object TTControllers {

    fun connectFour(): TTController {
        return TTClassicControllerWithGravity(TTFactories().classicMNK(7, 6, 4))
    }

    fun classicTTT(): TTController {
        return TTClassicController(TTFactories().classicMNK(3))
    }

    fun ultimateTTT(): TTController {
        return TTUltimateController(TTFactories().ultimate())
    }

}
